firstvoices/.coveragerc

```diff
@@ -1,13 +1,6 @@
 [run]
-branch = True
-relative_files = True
-include =
-    backend/*
-    firstvoices/*
-    healthcheck/*
-    jwt_auth/*
-    scripts/*
-    manage.py
+source=.
+branch=True
 omit =
     */tests/*
     */migrations/*
```

Comment:
Looks like this is still excluding the resources folder from back when we were focused on the initial data migrations. Now that we're using the resources for application code I think they should get included.

Not sure why the search tasks are also excluded-- can they be included as well or is there some problem?

I think the fixtures folder could be excluded though. It probably gets loaded by every test, but it isn't meaningful lines of code and will skew our percentages.